lib/macho/load_commands.rb

```diff
@@ -181,6 +209,22 @@ def initialize(view, cmd, cmdsize)
       @cmdsize = cmdsize
     end
 
+    # @return [Boolean] true if the load command can be serialized, false otherwise
+    def serializable?
+      CREATABLE_LOAD_COMMANDS.include?(LOAD_COMMANDS[cmd])
+    end
+
+    # @param context [MachO::LoadCommand::SerializationContext] the context
+    #  to serialize into
+    # @return [String, nil] the serialized fields of the load command, or nil
+    #  if the load command can't be serialized
+    # @api private
+    def serialize(context)
+      return unless serializable?
+      format = Utils.specialize_format(FORMAT, context.endianness)
+      [cmd, cmdsize].pack(format)
```

Comment:
Is this intended as a generic implementation that works for load commands that only contain a fixed set of fields and have a fixed size? If so, `cmdsize` still needs to be populated with `SIZEOF` (or simply replaced with it in this line).